CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 3.10
+
+* Update Python analyzer 3.15 -> [3.16](https://github.com/SonarSource/sonar-python/releases/tag/3.16.0.9967), 8 new unit test rules
+* Update Secrets analyzer 1.1 -> [1.2](https://sonarsource.atlassian.net/browse/SECRETS-20?jql=project%20%3D%20SECRETS%20AND%20fixVersion%20%3D%201.2), remove warning about packaged dependencies:
```

Comment:
```suggestion
* Update Secrets analyzer 1.1 -> [1.2](https://sonarsource.atlassian.net/secure/ReleaseNote.jspa?projectId=10101&version=11449), remove warning about packaged dependencies
```

Reply:
There is a new guide for using Jira release notes - see [document](https://docs.google.com/document/d/1pYOAGJ7TwmEVLE9U48LivjwWhyskDrFBD88LQPesiiU/edit). We should use the `Issue Navigator` link, instead of actual release notes. I also don't fully understand the reason behind tbh, but the whole company is doing the new thing

Reply:
`Request for Release` chapter of [this page](https://xtranet-sonarsource.atlassian.net/wiki/spaces/DEV/pages/776802/Release+Procedures)

Reply:
Thanks for the pointers to this new documentation.

In this case, I believe we should remove the link altogether - it's not accessible to people outside the company anyway.

Reply:
Makes sense, removed. Thanks for the review!